### PR TITLE
DEV-3519-feat(piscine-go): add README.md for new go exercise dealapackofcards

### PR DESCRIPTION
**Pull request by tiagocollot**
Created and added new go exercise dealapackofcards.

### CODE REVIEW

**Review by jotapero — APPROVED** [review]

File: subjects/dealapackofcards/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## dealapackofcards
+
+### Instructions
+
+Deal a pack of `12` cards evenly between 4 players, `Player 1`, `Player 2`, `Player 3` and `Player 4`.
+
+Write a function `DealAPackOfCards()` that takes an argument `deck` as an `int` and prints the desired output.
```
Comment (eslopfer) [inline subjects/dealapackofcards/README.md:7]: Slice of int

File: subjects/dealapackofcards/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+func main() {
+	deck := []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12}
+  	piscine.DealAPackOfCards(deck)
+}
+```
+
+And its output:
+
+```go
```
Comment (eslopfer) [inline subjects/dealapackofcards/README.md:36]: Console instead of go.

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: subjects/dealapackofcards/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/dealapackofcards/README.md]: It is fixed.

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**